test/helpers/policygen/models.go

```diff
@@ -791,7 +792,11 @@ func (tg TestSpecsGroup) CreateAndApplyManifests(kub *helpers.Kubectl) {
 	for _, test := range tg {
 		test.CreateManifests()
 		manifests = append(manifests, test.GetManifestsPath())
+		test.Kub = kub
+		err := test.Destination.CreateApplyManifest(test)
```

Comment:
What is the exact reasoning for this change? It is not clear.

Reply:
The main reason is to create pods and the given service at the same time. It didn't make sense to me to have a function called `ConnectivityTest`  that created some services. This is the way that we follow in Nightly.